catalog-info.yaml

```
@@ -0,0 +1,27 @@
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elasticsearch-k8s-metrics-adapter
+  description: Buildkite Pipeline for elasticsearch-k8s-metrics-adapter
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elasticsearch-k8s-metrics-adapter
+
+spec:
+  type: buildkite-pipeline
+  owner: group:cloud-k8s-operator
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elasticsearch-k8s-metrics-adapter
+    spec:
+      repository: elastic/elasticsearch-k8s-metrics-adapter
+      pipeline_file: ".buildkite/pipeline.yml"
+      teams:
+        cloud-k8s-operator: {}
```

We probably want the same config than https://github.com/elastic/lightweight-cert-manager/blob/15a35783b13d1a7b54aa901c65e931b4246cacd6/catalog-info.yaml#L47-L49:

```suggestion
        cloud-k8s-region:
          access_level: MANAGE_BUILD_AND_READ
```